code/game/objects/items/weapons/autolathe_disk/absolute_disks.dm

```diff
@@ -54,85 +52,14 @@
 		/datum/design/bioprinter/pouch/medium_generic,
 		/datum/design/bioprinter/pouch/large_generic,
 		/datum/design/bioprinter/pouch/tubular,
-		/datum/design/bioprinter/pouch/tubular/vial,
-		/datum/design/bioprinter/sling,
-		//power cells
-		/datum/design/bioprinter/nt_cells/large,
-		/datum/design/bioprinter/nt_cells/large/plasma,
-		/datum/design/bioprinter/nt_cells/medium,
-		/datum/design/bioprinter/nt_cells/medium/plasma,
-		/datum/design/bioprinter/nt_cells/small,
-		/datum/design/bioprinter/nt_cells/small/plasma,
-		//utilities
-		/datum/design/autolathe/gun/nt_sprayer,
-		/datum/design/autolathe/device/grenade/nt_cleaner,
-		/datum/design/autolathe/device/grenade/nt_weedkiller,
-		/datum/design/bioprinter/holyvacuum
+		/datum/design/bioprinter/pouch/tubular/vial
 	)
 
-
 /obj/item/computer_hardware/hard_drive/portable/design/nt_bioprinter
 	disk_name = "Church of Absolute Bioprinter Products and Utilities"
 	desc = "This disk is sole property of the church, the files within are encrypted and should not be used, taken, or tested by anyone not affiliated with the Absolute."
 	icon_state = "neotheology_testament_u"
-
 	license = 20
-	designs = list(
```

Comment:
Did you mean to make this dis have 0 designs?

Reply:
Shouldn't it inherit designs from the base disk just have license costs ?

Reply:
What base designs?

Reply:
The pathing was all weird so now it should be good

Reply:
Looks good